apps/templates/content_templates_test.py

```diff
@@ -76,8 +76,7 @@ def test_render_content_template(self):
         template = {
             '_id': 'foo',
             'template_name': 'test',
-            'template_desk': 'sports',
-            'template_stage': 'schedule',
+            'template_desks': ['sports'],
```

Comment:
Or alternatively a behave test case for "creating story out of a template that has multiple desks land on ... desk" would be great.